<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auth</class>
 <widget class="QWidget" name="Auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Auth</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(26, 22, 31);
}

QTabBar::tab { 
	border: none;
	width: 140px;
	height: 40px;
	font: 500 14pt &quot;Archivo Medium&quot;;
}

QTabBar::tab:selected{
	border-bottom: 2px solid rgb(17, 57, 149);
}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>280</width>
     <height>440</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget {
	border: none;
}
QPushButton {
	background-color: rgb(17, 57, 149);
	color: rgb(255, 255, 255);
	border-radius: 15px;
	font: 500 14pt &quot;Archivo Medium&quot;;
}
QPushButton:hover {
	font: bold;
	background-color: rgb(102, 128, 192);
	color: black;
}
QPushButton:pressed {
	background-color: rgb(85, 85, 0);
	color: rgb(255, 255, 255);
	background-color: rgb(102, 128, 192);
}
QLineEdit {
	border: none;
	border-bottom: 2px solid rgb(239, 239, 239);
	color: rgb(239, 239, 239);
	padding-bottom: 7px;
	font: 500 12pt &quot;Archivo Medium&quot;;
}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tab_sing_in">
    <attribute name="title">
     <string>Sing in</string>
    </attribute>
    <widget class="QLineEdit" name="username_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string> User Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string> Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="error_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
font: 500 10pt &quot;Archivo Medium&quot;;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Error messege</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="sing_in">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>280</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>SING IN</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_sing_up">
    <attribute name="title">
     <string>Sing up</string>
    </attribute>
    <widget class="QLineEdit" name="password_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string> Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="error_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
font: 500 10pt &quot;Archivo Medium&quot;;</string>
     </property>
     <property name="text">
      <string>Error messege</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usename_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="placeholderText">
      <string> User Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sing_up">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>280</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>SING UP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="confirm_password">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string> Confirm Password</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/movie_icon_133893.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icon/res.qrc"/>
 </resources>
 <connections/>
</ui>
